app/components/find/results/results_component.html.erb

```diff
@@ -26,7 +26,8 @@
           <%= render Find::Results::SearchResultComponent.new(
             course:,
             has_sites: results.has_sites?(course),
-            filtered_by_location: results.location_filter?
+            filtered_by_location: results.location_filter?,
+            sites_count: results.sites_count(course)
```

Comment:
Do we need `has_sites` and `sites_count` if `sites_count` is more than `0` we know it `has_sites` 🎉

Reply:
I think the issue is that I need to know if there are more than 1 sites, not 0. https://github.com/DFE-Digital/publish-teacher-training/blob/214273b8f801c680f6a5332248a87e621dd59ac2/app/components/find/results/search_result_component.rb#L29